etc/example-config.yaml

```diff
@@ -35,3 +35,5 @@ broker:
   launch_apb_on_bind: false
   recovery: true
   output_request: true
+  ssl_cert_key: /path/to/key
```

Comment:
IIRC when we're using the broker in dev mode we should skip ssl and run insecure.  But it looks like you covered that with the insecure flag so really what goes here won't matter if we use that flag.